src/Illuminate/Hashing/ArgonHasher.php

```diff
@@ -60,13 +61,13 @@ public function __construct(array $options = [])
      */
     public function make(#[\SensitiveParameter] $value, array $options = [])
     {
-        $hash = @password_hash($value, $this->algorithm(), [
-            'memory_cost' => $this->memory($options),
-            'time_cost' => $this->time($options),
-            'threads' => $this->threads($options),
-        ]);
-
-        if (! is_string($hash)) {
+        try {
+            $hash = password_hash($value, $this->algorithm(), [
+                'memory_cost' => $this->memory($options),
+                'time_cost' => $this->time($options),
+                'threads' => $this->threads($options),
+            ]);
+        } catch (Error) {
```

Comment:
Might it be helpful to add this caught Error as the `previous` parameter of the RuntimeException?

Reply:
yah, I was considering that. is there any security implication though, because it could expose info like which algo is being used?

Reply:
Isn't it already showing the hashing algo in the exception message?

In theory, these exceptions should not be passed along to the end-user when the app is set to production mode.

(I don't have a strong opinion on it either way, just a thought as I have fallen into the practice of forwarding exceptions when throwing a new one)

Reply:
It also catches a generic Error object, which could be caused by any number of things. At least this way you can see what specifically happened.

Reply:
it could also expose other info, like cost factor, etc. 

you're right, it should never show to users in production.

I'd say give a PR a shot, and see what other people think.